human_body_prior/body_model/body_model.py

```diff
@@ -63,6 +63,8 @@ def __init__(self,
         # -- Load SMPL params --
         if '.npz' in bm_path:
             smpl_dict = np.load(bm_path, encoding='latin1')
+        elif '.pkl' in bm_path:
+            smpl_dict = np.load(bm_path, allow_pickle = True)
```

Comment:
with numpy==1.19.5 it was neccessary to also use encoding like
smpl_dict = np.load(bm_path, allow_pickle = True, encoding='latin1')